crates/sui-core/src/validator_client_monitor/mod.rs

```diff
@@ -45,6 +45,8 @@ pub struct OperationFeedback {
     pub display_name: String,
     /// The operation type
     pub operation: OperationType,
+    /// The ping type. If it's not a ping request, then this is is None.
+    pub ping: Option<PingType>,
```

Comment:
It seems we only use if this request is part of a ping or not. How about using `ping: bool` instead?